src/main/scala/org/apache/spark/deploy/ArmadaSparkSubmit.scala

```diff
@@ -0,0 +1,1276 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.deploy
+
+// Initially copied from: https://raw.githubusercontent.com/apache/spark/9cf98ed41b2de1b44c44f0b4d1273d46761459fe/core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala
```

Comment:
I presume the following classes (those containing un-commented code) are WIP and subject to be refined next.

Reply:
Yes, as I mentioned above, this PR is just meant to move the code over to the new repo, and get it working (without changing too much.)

We will need to refine the commented out code in future PR's.  I'll create a separate ticket for that.

Reply:
https://github.com/G-Research/spark/issues/59